internal/services/integrationtesting/consistency_test.go

```diff
@@ -411,6 +418,31 @@ func validateLookupResources(t *testing.T, vctx validationContext) {
 									expectedPermissionship,
 									permissionship,
 								)
+
+								bulkCheckItems = append(bulkCheckItems, &v1.BulkCheckPermissionRequestItem{
+									Resource: &v1.ObjectReference{
+										ObjectType: resourceRelation.Namespace,
+										ObjectId:   resolvedResource.ResourceObjectId,
+									},
+									Permission: resourceRelation.Relation,
+									Subject: &v1.SubjectReference{
+										Object: &v1.ObjectReference{
+											ObjectType: subject.Namespace,
+											ObjectId:   subject.ObjectId,
+										},
+										OptionalRelation: stringz.Default(subject.Relation, "", tuple.Ellipsis),
```

Comment:
there's a `DefaultEmpty` for this specific use case, haha

Reply:
I copied this from elsewhere in our code